### DISCUSSION TITLE
Elementary OS 7 Horus - LUKS Volumes support (disk encryption)

### DISCUSSION THREAD

**Discussion by mdz1998** [elementary/triage#279] [Q&A]
I am struggling with setting up LUKS Volumes with encryption for Elementary OS 7 Horus.

I am using this tutorial: https://www.mikekasberg.com/blog/2020/04/08/dual-boot-ubuntu-and-windows-with-encryption.html

Unfortunately Elementary OS installer does not support LUKS volumes.  I cannot use any of LUKS volumes/partition I have created.

So the question is: How Can I install Elementary os with the LUKS volumes? Or in other words: How I can set up Disk encryption on Elementary OS (installed along with dual boot with Windows 10)?

![pP2D7](https://github.com/elementary/triage/assets/29581005/e777f69c-29c9-480c-9dba-4f3960d9635f)

**Comment by lenemter** [comment]
I think this is not currently supported. See https://github.com/elementary/installer/issues/619. There is a link to a tutorial how to setup encription, but seems like it requires reinstalling Windows :( https://github.com/elementary/installer/issues/619#issuecomment-980565905

**Comment by lenemter** [comment]
I'll close this as duplicate of https://github.com/elementary/installer/issues/619
